doc_template/install.rst

```diff
@@ -71,22 +71,25 @@ example Dockerfiles are available.
 
  #. Ensure you have Docker installed.
 
- #. Use Docker to build one of the images.
- 
-     Anaconda::
+ #. Use Docker to build the image::
 
-         docker pull alleninstitute/allensdk
+     docker pull alleninstitute/allensdk
  
-     Other docker configurations are also available under docker directory in the source repository.
+    Other docker configurations are also available under docker directory in the source repository.
  
  #. Run the docker image::
  
-     docker run -i -t -p 8888:8888 -v /data:/data alleninstitute/allensdk /bin/bash
+     docker run -i -t -p 8888:8888 alleninstitute/allensdk /bin/bash
+
+ #. Run the SDK tests::
+
      cd allensdk
      make test
  
  #. Start a Jupyter Notebook::
  
      cd allensdk/doc_template/examples_root/examples/nb
-     jupyter-notebook --ip=* --no-browser
+     jupyter notebook --ip=* --no-browser --allow-root
```

Comment:
If you update the docker file to have WORKDIR not at root like in my other comment I don't think you'll need to specify --allow-root.

Reply:
Unfortunately I tried, but it still threw the error message. I think that's the only user that comes with the continuumio/anaconda3 image.

Reply:
Unfortunate, thanks for checking on this!